packages/react-scripts/package.json

```diff
@@ -1,33 +1,33 @@
 {
-  "name": "react-scripts",
-  "version": "5.0.0",
-  "description": "Configuration and scripts for Create React App.",
+  "name": "@skyscanner/backpack-react-scripts",
+  "version": "10.0.1",
```

Comment:
should be 11.0.0 ?

Reply:
Yes, does Changelog need to add upgrade information now?

Reply:
yes, please add

Reply:
Ok